.github/CODEOWNERS

```diff
@@ -4,55 +4,56 @@
-/plugins/kafka-backend                              @backstage/reviewers @nirga
-/plugins/newrelic-dashboard                         @backstage/reviewers @mufaddal7
-/plugins/scaffolder-backend-module-rails            @backstage/reviewers @angeliski
-/plugins/scaffolder-backend-module-yeoman           @backstage/reviewers @pawelmitka
-/plugins/search                                     @backstage/reviewers @backstage/techdocs-core
-/plugins/search-*                                   @backstage/reviewers @backstage/techdocs-core
-/plugins/stack-overflow                             @backstage/reviewers @backstage/techdocs-core
-/plugins/stack-overflow-backend                     @backstage/reviewers @backstage/techdocs-core
-/plugins/sonarqube                                  @backstage/reviewers @backstage/sda-se-reviewers
-/plugins/techdocs                                   @backstage/reviewers @backstage/techdocs-core
-/plugins/techdocs-*                                 @backstage/reviewers @backstage/techdocs-core
-/tech-insights-backend                              @backstage/reviewers @xantier @iain-b
-/tech-insights-backend-module-jsonfc                @backstage/reviewers @xantier @iain-b
-/tech-insights-tech-insights-common                 @backstage/reviewers @xantier @iain-b
-/tech-insights-tech-insights-node                   @backstage/reviewers @xantier @iain-b
+*                                                 @backstage/maintainers
+*/yarn.lock                                       @backstage/reviewers @backstage-service
+/.changeset/cost-insights-*                       @backstage/reviewers @backstage/silver-lining
+/.changeset/search-*                              @backstage/reviewers @backstage/techdocs-core
+/.changeset/techdocs-*                            @backstage/reviewers @backstage/techdocs-core
+/cypress/src/integration/plugins/techdocs.spec.ts @backstage/reviewers @backstage/techdocs-core
+/docs/assets/search                               @backstage/reviewers @backstage/techdocs-core
+/docs/features/search                             @backstage/reviewers @backstage/techdocs-core
+/docs/features/techdocs                           @backstage/reviewers @backstage/techdocs-core
+/packages/search-common                           @backstage/reviewers @backstage/techdocs-core
+/packages/techdocs-cli                            @backstage/reviewers @backstage/techdocs-core
+/packages/techdocs-cli-embedded-app               @backstage/reviewers @backstage/techdocs-core
+/packages/techdocs-common                         @backstage/reviewers @backstage/techdocs-core
+/plugins/allure                                   @backstage/reviewers @deepak-bhardwaj-ps
+/plugins/apache-airflow                           @backstage/reviewers @cmpadden
+/plugins/api-docs                                 @backstage/reviewers @backstage/sda-se-reviewers
+/plugins/apollo-explorer                          @apollographql/solutions
```

Comment:
This doesn't work afaik, teams need to exist within this org. Would need to list out the users instead. Please include `@backstage/reviewers` as well